.gitignore

```diff
@@ -1,22 +1,45 @@
-.DS_Store
```

Comment:
This seems a bit like PR pollution - is it necessary to change all of this?

Reply:
yes, i got carried away with some cleanup, sorry.